interactions/api/models/channel.py

```diff
@@ -317,6 +317,20 @@ async def modify(
         )
         return Channel(**res, _client=self._client)
 
+    async def add_member(
+        self,
+        member_id: int,
+    ) -> None:
+        """
+        This adds a member to the channel, if the channel is a thread
+
+        :param member_id: The id of the member to add to the channel
+        :type member_id: int
+        """
+        if not self.thread_metadata:
+            raise TypeError("The Channel you specified is not a thread!")
```

Comment:
```suggestion
            raise TypeError("The Channel you specified is not a thread!")  # TODO: Move to new error formatter.
```

Adds an internal note for the developers to flag for 2022 roadmap work.